docs/guides/earning-cred.md

```diff
@@ -1,5 +1,34 @@
 ---
 title: 💧 Earning Cred
+description: How to get Cred from your contributions.
 ---
 
-TODO
+As soon as you interact with Sourcecred, you will be part of the Contribution Graph (i.e. a network of connected contributions) in which some [Cred] will flow or be minted in your direction. 
+
+
+Your contributions is tracked across the three active plugins we have for now. The types of contributions & plugins are likely to increase. But for now, small is beautiful.
+
+
+| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]|
+| -- | -- | -- |
+|someone gives your post : :sourcecred:  or :100: or :+1: emoji | someone gives your post/reply a :heart: | You open a pull request  |
+
+
+
+
+### 💧 Cred Minting
+the community decides to Mint some Cred for you, e.g. because they found something valuable. This Cred just magically appears out of nowhere in your direction. Say differently, you are now able to collect some drops of water from the sky
+
+
+
+### 🏄🏾Cred flowing
+Being part of the community, you share the same Contribution Graph. Your contribution is interconnected with others in all its different ways and across the time. Someone has responsed to your post. This same person has liked some other posts. Or later, another person will acknowledge your contribution. Cred will be flowing in/out on you but at a different speed. 
```

Comment:
Again, I really like the meaning behind this, and want to make it a bit clearer. How do you feel about this rewording?
"All the contributions you and your community make are on the same Contribution Graph. Each of your contributions is interconnected with other community contributions, even across different platforms. They even also through time - today's contributions connect to contributions from yesterday, last week, etc."